README.md

```diff
@@ -1,2 +1,167 @@
-# template
-Use this template for web standards proposals in this org.
+# Explainer for the TODO API
+
+**Instructions for the explainer author: Search for "todo" in this repository and update all the
+instances as appropriate. For the instances in `index.bs`, update the repository name, but you can
+leave the rest until you start the specification. Then delete this block of text.**
+
+This proposal is an early design sketch by [TODO: team] to describe the problem below and solicit
+feedback on the proposed solution. It has not been approved to ship in Chrome.
+
+## Proponents
+
+- [Proponent team 1]
```

Comment:
Should these have "todo" on them, or should the instructions say to look for brackets?

Reply:
I hope moving the "TODO: Fill in the whole explainer template" bit above this is sufficient.